### PR TITLE
Add net.bartkessels.getit

### PR DESCRIPTION
**Pull request by bartkessels**
A GTK HTTP request application written in C for the Gnome desktop environment.

### CODE REVIEW

**Comment by nedrichards** [comment]
net.bartkessels.getit

File: net.bartkessels.getit.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
+{
+	"app-id": "net.bartkessels.getit",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.24",
```
Comment (nedrichards) [inline net.bartkessels.getit.json]: Does it specifically require something from 3.24 or is that just what it was tested on?

File: net.bartkessels.getit.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	"sdk": "org.gnome.Sdk",
+	"command": "getit",
+	"finish-args": [
+		"--socket=x11",
+		"--share=ipc",
+		"--socket=wayland",
+		"--share=network",
+		"--filesystem=home",
```
Comment (nedrichards) [inline net.bartkessels.getit.json]: What do you need full filesystem access for? If you use GtkFileChooserNative https://developer.gnome.org/gtk3/stable/gtk3-GtkFileChooserNative.html then your app automatically uses the portal to get only the files you user asks for.

File: net.bartkessels.getit.json [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+	],
+	"modules": [
+		{
+			"name": "gtksourceview-3.0",
+			"config-opts": ["--disable-Werror"],
+			"builddir": true,
+			"sources": [
+				{
+					"type": "git",
+					"branch": "gnome-3-24",
```
Comment (nedrichards) [inline net.bartkessels.getit.json]: this isn't specific enough as a release. it needs a tag or commit as well - or you could just use the archive, it's usually faster.

File: net.bartkessels.getit.json [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+		},
+		{
+			"name": "getit",
+			"buildsystem": "meson",
+			"builddir": true,
+			"sources": [
+				{
+					"type": "git",
+					"branch": "stable",
+					"url": "https://github.com/bartkessels/getit"
```
Comment (nedrichards) [inline net.bartkessels.getit.json]: this needs a specific release rather than just a branch.

**Comment by nedrichards** [comment]
bot, build net.bartkessels.getit

File: net.bartkessels.getit.json (hunk 1 above)
Comment (bartkessels) [reply net.bartkessels.getit.json]: No, it doesn't require anything specific from 3.24 it's only tested with that version

**Comment by bartkessels** [comment]
I've updated the application and flatpak JSON file to conform to your suggestions

**Comment by nedrichards** [comment]
bot, build net.bartkessels.getit

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/net.bartkessels.getit

**Pull request closed without merging**